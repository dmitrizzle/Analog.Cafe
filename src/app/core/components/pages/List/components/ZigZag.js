import React from "react"

// return
export default () => {
  return (
    <svg
      viewBox="0 0 500 1000"
      xmlns="http://www.w3.org/2000/svg"
      preserveAspectRatio="none slice"
    >
      <path
        d="M 4 1001 L 0 1001 L 0 -1 L 4 -1 L 169 343 L 4 1001 Z"
        fill="#ffffff"
      />
    </svg>
  )
}
