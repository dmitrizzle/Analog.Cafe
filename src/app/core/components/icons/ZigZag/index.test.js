import "jest-styled-components"

import { shallow } from "enzyme"
import React from "react"

import ZigZag from "./"

it("Render ZigZag, matches snapshot", () => {
  const element = shallow(<ZigZag />)
  expect(element).toMatchSnapshot()
})
