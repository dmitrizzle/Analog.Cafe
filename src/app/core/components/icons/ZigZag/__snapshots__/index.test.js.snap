// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render ZigZag, matches snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <path
        d="M 4 1001 L 0 1001 L 0 -1 L 4 -1 L 169 343 L 4 1001 Z"
        fill="#ffffff"
/>,
      "preserveAspectRatio": "none slice",
      "viewBox": "0 0 500 1000",
      "xmlns": "http://www.w3.org/2000/svg",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "d": "M 4 1001 L 0 1001 L 0 -1 L 4 -1 L 169 343 L 4 1001 Z",
        "fill": "#ffffff",
      },
      "ref": null,
      "rendered": null,
      "type": "path",
    },
    "type": "svg",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <path
          d="M 4 1001 L 0 1001 L 0 -1 L 4 -1 L 169 343 L 4 1001 Z"
          fill="#ffffff"
/>,
        "preserveAspectRatio": "none slice",
        "viewBox": "0 0 500 1000",
        "xmlns": "http://www.w3.org/2000/svg",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "d": "M 4 1001 L 0 1001 L 0 -1 L 4 -1 L 169 343 L 4 1001 Z",
          "fill": "#ffffff",
        },
        "ref": null,
        "rendered": null,
        "type": "path",
      },
      "type": "svg",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
